Symfony/CS/Tests/Tokenizer/TokensTest.php

```diff
@@ -882,4 +882,128 @@ public function provideShortOpenTagEchoMonolithicPhpDetection()
             array("<?=' '\n?><?=' ';\n", false),
         );
     }
+
+    /**
+     * @dataProvider provideIsArray
+     * @requires PHP 5.4
+     *
+     * @param string $source   PHP code
+     * @param array  $expected [0] Token index to use,
```

Comment:
this does not correspond to the actual array. The token index is not the first element of the array. It is the key.